api/nakadi-event-bus-api.yaml

```diff
@@ -892,12 +905,18 @@ definitions:
       oldest_available_offset:
         description: |
           An offset of the oldest available Event in that partition. This value will be changing upon
-          removal of Events from the partition by the background archiving/cleanup mechanizms.
+          removal of Events from the partition by the background archiving/cleanup mechanism.
         type: string
       newest_available_offset:
         description: |
-          An offset of the newes available Event in that partition. This value will be changing upon
-          reception of new events for this partition by nakadi
+          An offset of the newest available Event in that partition. This value will be changing upon
+          reception of new events for this partition by Nakadi.
+
+          This value can be used to construct a cursor when opening streams (see
+          `GET /event-type/{name}/events` for details).
+
+          Might assume the special name OLDEST, meaning a pointer to the offset of the oldest available
+          event in the partition.
```

Comment:
In my understanding it's one event BEFORE the oldest available. Hence, the term "OLDEST" is still misleading although the effect will be what you want. When consuming "OLDEST" makes sense, when exposing something to the client it leads to confusion.